core/src/main/java/tc/oc/pgm/regions/RegionMatchModule.java

```diff
@@ -413,30 +409,34 @@ private static Material getHangingType(Hanging hanging) {
     }
   }
 
+  protected static boolean processQuery(RegionFilterApplication rfa, Query query) {
+    return processQuery(query.getEvent(), rfa, query);
+  }
+
   /**
    * Query the RFA's filter with the given objects. If the query is denied, cancel the event and set
    * the deny message. If the query is allowed, un-cancel the event. If the query abstains, do
    * nothing.
    *
    * @return false if the query abstained, otherwise true
    */
-  protected static boolean processQuery(RegionFilterApplication rfa, Query query) {
+  protected static boolean processQuery(Event event, RegionFilterApplication rfa, Query query) {
     if (rfa.filter == null) {
       return false;
     }
 
     switch (rfa.filter.query(query)) {
```

Comment:
The query being used does not have the event, meaning, again, cause filters will not work